source/backend/tests/unit/api/Controllers/Activities/AcquisitionControllerTest.cs

```diff
@@ -334,7 +334,7 @@ public void UpdateActivityProperty_BadRequest()
             service.Setup(m => m.UpdateActivityAcquisitionProperties(It.IsAny<PimsActivityInstance>())).Returns(activity);
 
             // Act
-            Action act =  () => controller.UpdateActivityProperties(FileType.Unknown, mapper.Map<ActivityInstanceModel>(activity));
+            Action act = () => controller.UpdateActivityProperties(FileType.Unknown, mapper.Map<ActivityInstanceModel>(activity));
```

Comment:
Since you are modifying this file - could you please rename it to match the class name?
![image](https://user-images.githubusercontent.com/24322224/195190946-e2f57ebd-6c6d-4cf8-95b5-01f12dc67768.png)